hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.test.tsx

```diff
@@ -15,7 +24,40 @@ test("renders without crashing", () => {
         },
     } as RouteComponentProps<PageParams>;
     render(<TeamDetail {...teamDetailProps} />);
+
+    expect(screen.getByTestId("team-info-linear-progress")).toBeInTheDocument();
+
     expect(
         screen.getByText(`Team ${teamDetailProps.match.params.id} Overview`)
     ).toBeInTheDocument();
+
+    expect(mockedGet).toHaveBeenCalledWith(`/api/event/teams/${mockTeamMultiple.id}/`);
+});
+
+test("displays 404 error when the requested team id is not found", async () => {
+    const failureResponse = {
+        response: {
+            status: 404,
+            statusText: "Not Found",
+            message: "Could not find team code: Error 404",
+        },
+    };
+
+    mockedGet.mockRejectedValueOnce(failureResponse);
+
+    const teamDetailProps = {
```

Comment:
Since you're using teamDetailProps in both unit tests, you can turn this into a global constant.

Reply:
yee